float64 time
int32 n_samples
int32 n_channels
int32    n_frequency
int16[] l_channel_data
int16[] r_channel_data

